src/ucp/core/ucp_listener.c

```diff
@@ -77,6 +77,12 @@ ucs_status_t ucp_conn_request_query(ucp_conn_request_h conn_request,
         }
     }
 
+    if ((attr->field_mask & UCP_CONN_REQUEST_ATTR_FIELD_WORKER_ID)) {
+        /* coverity[overrun-local] */
+        attr->worker_id = ucp_address_get_uuid(
+            UCS_PTR_TYPE_OFFSET(&conn_request->sa_data, ucp_wireup_sockaddr_data_t));
+    }
```

Comment:
what if worker uuid is not present? need to return error or some default value

Reply:
It will return 0 if WORKER_UID flag not present in address: https://github.com/openucx/ucx/pull/7237/files#diff-ad67b273069708eb30c90a60c2b6e69696065480ba4413348289e834836c59f2R627